stdlib/cmake/modules/SwiftSource.cmake

```diff
@@ -566,17 +550,11 @@ function(_compile_swift_files
     list(APPEND module_outputs "${interface_file}")
   endif()
 
-  if(SWIFTFILE_SDK IN_LIST SWIFT_APPLE_PLATFORMS)
```

Comment:
In theory, this is a change of behavior for Mac Catalyst, but I don't see any signs that this particular path was ever exercised by Catalyst libraries.

Reply:
This was purely a mechanical unconditonalization of the paths.  I was aiming to do the minimal thing to make this work.  There were a few paths that were of questionable value, but this isn't about the cleanup to the build, merely trying to get the same structure on all the targets.